optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/SolverBenchmarkResult.java

```diff
@@ -359,6 +361,10 @@ protected void determineStandardDeviation() {
         }
     }
 
+    protected EnvironmentMode determineEnvironmentMode() {
+        return ObjectUtils.defaultIfNull(solverConfig.getEnvironmentMode(), EnvironmentMode.REPRODUCIBLE);
```

Comment:
This duplicates SolverConfig line 172: if the default ever changes there (to PRODUCTION for example), then this one would have to change too and the developer might miss that.
I'd rather see the  determineEnvironmentMode() method on solverconfig and make line 172 use that as well as this method (which I would name getEnvironmentMode(), even though it delgates to determineEnvironmentMode() because on Config objects the getters/setters must be simple (for XStream, JAXB IIRC), but on Result objects they don't need to be simple)